### PR TITLE
Added Russian translation for missing phrases (2.1.1)

### PR DESCRIPTION
**Pull request by andrey-helldar**
# Description

# Screenshots/Recordings/Graphs

### CODE REVIEW

**Review by SoftFever — APPROVED** [review]
Thank you

**Review by SoftFever — CHANGES_REQUESTED** [review]
Can you take a look at the errors reported by the CICD?

**Comment by andrey-helldar** [comment]
@SoftFever Yeah, no problem. The tests didn't run before. I'll take a look at them now and fix them.

File: localization/i18n/ru/OrcaSlicer_ru.po [hunk 1]
```diff
@@ -13764,8 +14062,8 @@ msgstr ""
 "органический). В то время как гибридный стиль создаёт структуру, схожую с "
 "обычную поддержкой при больших плоских нависаниях."
 
-msgid "Default (Grid/Organic"
-msgstr ""
+msgid "Default (Grid/Organic)"
```
Comment (andrey-helldar) [inline localization/i18n/ru/OrcaSlicer_ru.po:14065]:
```suggestion
msgid "Default (Grid/Organic"
```

Reverted the change of the key by which the translation is searched.

Comment (andrey-helldar) [reply localization/i18n/ru/OrcaSlicer_ru.po]: I'll leave it as undecided so that this change can be seen.

**Comment by SoftFever** [comment]
Thanks @andrey-helldar 
Seems there are some errors remaining.

Unfortunately, the check won't run automatically for the first time contributor, I have to manually trigger it.

File: localization/i18n/ru/OrcaSlicer_ru.po [hunk 2]
```diff
@@ -10963,9 +11118,27 @@ msgid ""
 "and for when tool changing.\n"
 "\n"
 msgstr ""
```
Comment (andrey-helldar) [inline localization/i18n/ru/OrcaSlicer_ru.po:11120]: @SoftFever I fixed two of the bugs, but there was a problem with the third. I have reconsidered the formation of the string itself and I don't understand what it is complaining about:

> ./localization/i18n/ru/OrcaSlicer_ru.po:11120: 'msgstr' is not a valid C format string, unlike 'msgid'. Reason: The character that terminates the directive number 1 is not a valid conversion specifier.

In this place, the only thing to pick on is the “%” symbol. But it occurs once, just like in the original text.

I also compared its occurrence with other translation keys, as well as the `c-format` format, and I don't understand what could be the reason. Any ideas how to fix it?

Comment (andrey-helldar) [reply localization/i18n/ru/OrcaSlicer_ru.po:11119]: I've removed the format checking, but I don't know if that's the right thing to do. Although after that the tests passed successfully.

image

Comment (SoftFever) [reply localization/i18n/ru/OrcaSlicer_ru.po:11119]: This change is fine.
Sometimes the get text will confuse.

**Review by SoftFever — APPROVED** [review]
Look good
Thank you!

File: localization/i18n/ru/OrcaSlicer_ru.po (hunk 2 above)
Comment (andrey-helldar) [reply localization/i18n/ru/OrcaSlicer_ru.po:11119]: Got it 👍

**Pull request merged**